gamemodes/husklesph/gamemode/sv_awards.lua

```diff
@@ -0,0 +1,133 @@
+-- last prop death award
+local function lastPropStandingCalcuation()
+	if GAMEMODE.LastRoundResult == 2 then
+		return GAMEMODE.LastPropDeath
+	end
+
+	return nil
+end
+
+
+-- get prop with least movement
+local function leastMovementCalcuation()
+	local minPly
+
+	for _, ply in ipairs(GAMEMODE:GetPlayingPlayers()) do
+		if ply:Team() != 3 then continue end
```

Comment:
Definitely these magic numbers are only here because I'm not sure if there's anything else to do with the current iteration of the Utility PR. I suppose it would be a good idea to finish #11 and get that merged in before this PR.

Reply:
I entirely thought we had already finished that PR, oops...